miner/block_builder.go

```diff
@@ -365,7 +372,23 @@ func (t *BlockBuilder) handleGossipAtx(data service.GossipMessage) {
 	}
 	atx.CalcAndSetId()
 
-	t.With().Info("got new ATX", log.AtxId(atx.ShortString()), log.LayerId(uint64(atx.PubLayerIdx)))
+	commitmentStr := "nil"
+	if atx.Commitment != nil {
+		commitmentStr = atx.Commitment.String()
+	}
+
+	t.With().Info("got new ATX",
+		log.String("sender_id", atx.NodeId.ShortString()),
+		log.AtxId(atx.ShortString()),
+		log.String("prev_atx_id", atx.PrevATXId.ShortString()),
+		log.String("post_atx_id", atx.PositioningAtx.ShortString()),
```

Comment:
```suggestion
		log.String("pos_atx_id", atx.PositioningAtx.ShortString()),
```
Should also fix the same field in `ATX Published`